packages/default/scss/fab/_theme.scss

```diff
@@ -85,6 +99,18 @@
         box-shadow: $fab-shadow, 0px 3px 4px rgba($fab-item-icon-bg, .4);
     }
 
+    // Disabled state
+    .k-fab-item.k-state-disabled,
+    .k-fab-item:disabled {
+        opacity: 1;
+
+        .k-fab-item-text,
+        .k-fab-item-icon {
+            color: tint( $fab-item-text, 6 );
```

Comment:
The text color in secondary, warning and light themeColor variants is sharper than before. Can we make it lighter? Like the color of the disabled item.

Reply:
we can tackle this at a later point